sdb/command.py

```diff
@@ -225,14 +234,145 @@ def __init_subclass__(cls, **kwargs):
 
     def _call(self,
               objs: Iterable[drgn.Object]) -> Optional[Iterable[drgn.Object]]:
-        # pylint: disable=missing-docstring
-        raise NotImplementedError
+        """
+        Optionally implemented by the user.
+
+        Commands that expect to manipulate and deal with any errors
+        of all the objects from the previous stage of the pipeline
+        passed as an iterable, should implement this method.
```

Comment:
I think it's not just error handling - e.g. they might want to maintain state between each object (to count them or summarize them in some way)